uber/templates/preregistration/preregbase.html

```diff
@@ -2,6 +2,14 @@
 {% block head_javascript %}
     {{ super() }}
     <script type="text/javascript">
+        {#
+          Make the PREREG_BADGE_TYPES array available in javascript. Also
+          converts the integer values to strings, because that is what we'll
```

Comment:
Just a note for anyone wondering what the heck is happening to this diff -- GitHub's yelling as us about opening what it thinks is a single-quote string and not closing it. The code is actually fine, we're just confusing the syntax parser by shoving Jinja2 code inside a javascript block.